.eslintrc.js

```diff
@@ -1,5 +1,10 @@
 module.exports = {
   // ...
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: "module",
+  },
```

Comment:
question: Why need this? What problem did it solve?

Reply:
@pkong-ds I get some parse error while moving `js` files to `src/pages`, and it's because missing parser config, maybe `eslint` not work for the files in `public` before?